### ISSUE TITLE
Error at npm run dev

### ISSUE THREAD

**Issue by jschilperoord** [ndelvalle/generator-api#6]
On node v4.4.7 with a generated app using defaults I get:

``` Javascript
[nodemon] 1.10.1
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `node index.js`
/Users/Jeffrey/Projects/test-api/node_modules/milieu/lib/milieu.js:4
let   ini;              // Lazy loaded
^^^

SyntaxError: Block-scoped declarations (let, const, function, class) not yet supported outside strict mode
    at exports.runInThisContext (vm.js:53:16)
    at Module._compile (module.js:373:25)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (/Users/Jeffrey/Projects/test-api/node_modules/milieu/index.js:2:16)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
```

Updating the eslintignore file doesn't seem to help. Can you help ?

**Comment by ndelvalle** [comment]
@jschilperoord check this closed issue #3 🙂

**Comment by jschilperoord** [comment]
Yikes! Sorry for missing that. Thanks for your prompt response 👍🏻
